### ISSUE TITLE
Add sample sizes for bins?

### ISSUE THREAD

**Issue by zmokhiber** [Economic/binipolate#2]
Was just talking to @metaCoop about this; would it be possible to have an option to add sample sizes to the command? would be useful for analyzing smaller data cuts.

**Comment by metaCoop** [comment]
Yes. Would be great to be able to report the bin sizes for each decile, especially when running cuts of subgroups.

**Comment by benzipperer** [comment]
just saw this - yes, this is a good idea and should be easy to implement. feel free to modify the code and submit a pull request.
